src/components/ThumbnailImage.js

```diff
@@ -19,26 +19,44 @@ const propTypes = {
     /** Do the urls require an authToken? */
     isAuthTokenRequired: PropTypes.bool.isRequired,
 
+    /** Width of the thumbnail image */
+    imageWidth: PropTypes.number,
+
+    /** Height of the thumbnail image */
+    imageHeight: PropTypes.number,
+
     ...windowDimensionsPropTypes,
 };
 
 const defaultProps = {
     style: {},
+    imageWidth: 200,
+    imageHeight: 200,
 };
 
 class ThumbnailImage extends PureComponent {
     constructor(props) {
         super(props);
 
         this.updateImageSize = this.updateImageSize.bind(this);
-
+        const {thumbnailWidth, thumbnailHeight} = this.calculateThumbnailImageSize(props.imageWidth, props.imageHeight);
         this.state = {
-            thumbnailWidth: 200,
-            thumbnailHeight: 200,
+            thumbnailWidth,
+            thumbnailHeight,
         };
     }
 
-    updateImageSize({width, height}) {
+    /**
+     * Compute the thumbnails width and height given original image dimensions.
+     *
+     * @param {Number} width - Width of the original image.
+     * @param {Number} height - Height of the original image.
+     * @returns {Object} - Object containing thumbnails width and height.
+     */
+
+    calculateThumbnailImageSize(width, height) {
+        if (!width || !height) { return {}; }
```

Comment:
Small nitpick but we usually avoid having if condition logic in a single line so can you change this to multi-line i.e.,
```
if (!width || !height) { 
    return {}; 
}
```

Reply:
Fixed.